lib/src/model/pdf_document_catalog.dart

```diff
@@ -79,4 +90,66 @@ class PDFDocumentCatalog {
     return Future.value(
         PDFPageObjectNode(_document, parent, _resolver, childObject));
   }
+
+  Future<List<PDFBookmark>> _readBookmarks(PDFObject outlines) async {
+    // get the pages root
+    PDFDictionary pagesRoot =
+        (await _resolver.resolve(_dictionary[const PDFName('Pages')]))!;
+
+    // get the pages kids array
+    PDFArray kidsArray =
```

Comment:
I feel like this code will only work if all pages are included in the root pages object. Which is not the case with tree-based  pages. Should we not use the `getPages` method to get the full page tree?